lib/keycloak.js

```diff
@@ -804,8 +804,8 @@ export default class Keycloak {
     async #processInit(initOptions) {
         const callback = this.#parseCallback(window.location.href);
 
-        if (callback) {
-            window.history.replaceState(window.history.state, '', /** @type {string} */ (callback.newUrl));
+        if (callback?.redirectUri) {
+            window.history.replaceState(window.history.state, '', callback.redirectUri);
```

Comment:
Wondering if the term `redirectUri` is ambiguous. But I believe it will in every case a redirect URI and never something else, so I think its good.

Reply:
Resolving this thread, I think this is sufficiently addressed